hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -354,18 +355,26 @@ class DashboardSystemStatus(t.TypedDict):
 
 
 def get_dashboard_hashes() -> DashboardCount:
-    return DashboardCount(
-        total=34217123456,
-        today=145609278,
-        updated_at="MockData and Timestamp",
-    )
+    return get_count_from_record_cls(PipelinePDQHashRecord)
 
 
 def get_dashboard_matches() -> DashboardCount:
+    return get_count_from_record_cls(PDQMatchRecord)
+
+
+def get_count_from_record_cls(
+    record_cls: t.Union[t.Type[PDQMatchRecord], t.Type[PipelinePDQHashRecord]]
```

Comment:
blocking: Can we make a real class `Record` that these inherit from? 

Then we could make `get_count_from_record_cls` a classmethod on `Record`

Reply:
The class exists will update.